persistent/Database/Persist/Quasi.hs

```diff
@@ -352,6 +352,17 @@ userAttrs = do
 -- [["sad"],["sogood"]]
 @
 
+== @!no-migrate@
```

Comment:
I picked this spot since it's right after the attribute docs. I could make it go one level deeper so it's nested within them :shrug: